lib/Data/Validate/Sanctions/Redis.pm

```diff
@@ -0,0 +1,230 @@
+package Data::Validate::Sanctions::Redis;
+
+use strict;
+use warnings;
+
+use parent 'Data::Validate::Sanctions';
+
+use Data::Validate::Sanctions::Fetcher;
+use Scalar::Util    qw(blessed);
+use List::Util      qw(max);
+use JSON::MaybeUTF8 qw(encode_json_utf8 decode_json_utf8);
+use Syntax::Keyword::Try;
+
+# VERSION
+
+sub new {
+    my ($class, %args) = @_;
+
+    my $self = {};
+    $self->{redis_read}  = $args{redis_read} or die 'Redis read connection is missing';
+    $self->{redis_write} = $args{redis_write};
```

Comment:
Can't we just have one redis connection (write)?

Reply:
I did it. Now there is just a single argument `connection`.